<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEDSHMainWindow</class>
 <widget class="QMainWindow" name="ZEDSHMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zinek Shader Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="ZEDSHEditorWidget" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="mnuRecentFiles">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="mnuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actLoadShader"/>
    <addaction name="actUploadToEngine"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="separator"/>
    <addaction name="actSelectAll"/>
    <addaction name="actDeselect"/>
    <addaction name="separator"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actDelete"/>
    <addaction name="separator"/>
    <addaction name="actFind"/>
    <addaction name="actReplace"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actCompile"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actPreference"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actOutput"/>
    <addaction name="actError"/>
    <addaction name="actDefinitions"/>
    <addaction name="actCompileParameters"/>
    <addaction name="actReflection"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actReplace">
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actUploadToEngine">
   <property name="text">
    <string>Upload To Engine</string>
   </property>
  </action>
  <action name="actPreference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actError">
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="actCompileParameters">
   <property name="text">
    <string>Compile Parameters</string>
   </property>
  </action>
  <action name="actReflection">
   <property name="text">
    <string>Reflection</string>
   </property>
  </action>
  <action name="actDefinitions">
   <property name="text">
    <string>Definitions</string>
   </property>
  </action>
  <action name="actLoadShader">
   <property name="text">
    <string>Load Shader</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actDeselect">
   <property name="text">
    <string>Deselect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZEDSHEditorWidget</class>
   <extends>QTextEdit</extends>
   <header>ZEDSHEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
